invenio_rdm_records/config.py

```diff
@@ -108,43 +110,6 @@ def _(x):
 allowed types and how this configuration variable is used.
 """
 
-RDM_RECORDS_CUSTOM_VOCABULARIES = {}
-"""Paths to custom controlled vocabularies.
-
-Of the shape:
-
-.. code-block:: python
-
-    {
-        '<dotted>.<path>.<to field1>': {
-            'path': '<absolute path to CSV file containing it>'
-        },
-        # ...
-        '<dotted>.<path>.<to fieldN>': {
-            'path': '<absolute path to CSV file containing it>'
-        }
-    }
-
-For example:
-
-.. code-block:: python
-
-    {
-        'resource_type': {
-            'path': os.path.join(
-                os.path.dirname(os.path.abspath(__file__)),
-                'my_resource_types.csv'
-            )
-        },
-        'contributors.role': {
-            'path': os.path.join(
-                os.path.dirname(os.path.abspath(__file__)),
-                'my_contributor_roles.csv'
-            )
-        }
-    }
-"""
-
 RDM_RECORDS_USER_FIXTURE_PASSWORDS = {
```

Comment:
shelvable: This was there before but... Why do we *only* allow the password through env var? e.g. in ES we allow it fully or partially. I'd say the consistent thing would be:
- Allow full configuration (not partial)
- If one wants to do it through yaml can (maybe their deployment can have the passwords secure somehow)
- If one wants to do all via env var, can do also

Reply:
Yeah, I guess it should just be part of the fixtures and not here.